log/log_test.go

```diff
@@ -61,6 +66,10 @@ func TestLoggerKit(t *testing.T) {
 			logging = kit.Warn
 		case Error:
 			logging = kit.Error
+		case All:
+			logging = kit.Info
+		case None:
+			logging = kit.Info
```

Comment:
none?

Reply:
There is an option to set logger to none. To test that scenario, I had to use a kit.Info as logging. You can check [here](https://github.com/Zondax/drand/blob/fbf0a1c3d16de44c2bfc0e7090d425adac967413/log/log_test.go#L43) the output is nil, as log leves is set to None